src/core/cache.ts

```diff
@@ -25,6 +25,7 @@ interface CacheEntries {
   onDemandChannels: string[];
   quoiFeurChannels: string[];
   recurringMessages: { id: string; channelId: string; frequency: Frequency; message: string }[];
+  score: { id: string; username: string; score: number }[];
```

Comment:
I think it's useless to store the `username` because it can change at any time. But you need to store the guild id. 

PS: name the key `scoreQuoiFeur` or something more specific

Reply:
Hey ! Thank for review, working on it 👯‍♂️